stock_picking_line_sequence/models/stock.py

```diff
@@ -34,25 +34,27 @@ def create(self, values):
 class StockMoveLine(models.Model):
     _inherit = "stock.move.line"
 
-    def _get_aggregated_product_quantities(self, **kwargs):
-        def get_aggregated_properties(move_line=False, move=False):
-            move = move or move_line.move_id
-            uom = move.product_uom or move_line.product_uom_id
-            name = move.product_id.display_name
-            description = move.description_picking
-            if description == name or description == move.product_id.name:
-                description = False
-            product = move.product_id
-            line_key = (
-                f"{product.id}_{product.display_name}_" f'{description or ""}_{uom.id}'
-            )
-            return line_key
+    def _get_aggregated_properties(self, move_line=False, move=False):
+        aggregated_properties = super()._get_aggregated_properties(move_line, move)
+        move = move or move_line.move_id
+        uom = move.product_uom or move_line.product_uom_id
+        name = move.product_id.display_name
+        description = move.description_picking
+        if description == name or description == move.product_id.name:
+            description = False
+        product = move.product_id
+        line_key = (
+            f"{product.id}_{product.display_name}_" f'{description or ""}_{uom.id}'
+        )
+        aggregated_properties["line_key"] = line_key
```

Comment:
@AungKoKoLin1997 As we just talked, can you please check and add a docstring to this method to explain why this override is necessary.

Reply:
@yostashiro  I misunderstood about it. We can get rid of extending this method.